pick_kernel/kernel.py

```diff
@@ -293,15 +332,16 @@ async def launch_kernel_with_parameters(self, stream, ident, parent, config):
         self._publish_status("idle")
 
     def parse_cell(self, cell):
-        if not cell.startswith("%%kernel.config"):
+        if not cell.startswith("%%kernel."):
```

Comment:
I could/should break the following changes off into another PR. It was just nice to have a fake namespace while working with `pick` right now.

Reply:
we can iterate on that over time for sure